app/src/main/java/ingsw/group1/findmyphone/location/LocationMessageParser.java

```diff
@@ -0,0 +1,105 @@
+package ingsw.group1.findmyphone.location;
+
+import android.location.Location;
+
+/**
+ * Parser class of static methods that provides composition of message for location request and response
+ * Message request is composed as:
+ * {@link LocationMessageParser#LOCATION_REQUEST_TAG}
+ * <p>
+ * Message response is composed in this way:
+ * {@link LocationMessageParser#LOCATION_RESPONSE_TAG}<LT>value_latitude</LT><LG>value_longitude</LG>
+ *
+ * @author Giorgia Bortoletti
+ */
+public class LocationMessageParser {
+
+    static final String LOCATION_REQUEST_TAG = "LOCATION_REQUEST";
+    static final String LOCATION_RESPONSE_TAG = "LOCATION_RESPONSE";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4929934)